atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/SweepStatsKeyValueService.java

```diff
@@ -262,6 +225,9 @@ private void flushWrites(Multiset<TableReference> writes, Set<TableReference> cl
             commit(timestamp);
             delegate().put(SWEEP_PRIORITY_TABLE, newWriteCounts, timestamp);
         } catch (RuntimeException e) {
+            if (Thread.interrupted()) {
```

Comment:
Should this be `if (Thread.currentThread().isInterrupted()) {` to avoid clearing interrupt flag?